### DISCUSSION TITLE
cron Automation trigger in non-prod

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#10629] [General]
I have a cron trigger based automation which is all working fine and in the prod/published application works as I expected and sends an email to all users in the users table.
However it also seems to run at the same time in the non-published side and sends again. 
Is there a way to prevent the non-published part from triggering?

**Comment by melohagan** [comment]
Hey @andyburgessmd 

Cron jobs shouldn't run in a non-published app. Are you cloud or self-hosted? 

Maybe a previous cron automation is hanging around incorrectly.
